src/file.hpp

```diff
@@ -113,7 +121,10 @@ namespace Sass {
 
   namespace File {
 
-    std::vector<Include> resolve_includes(const std::string& root, const std::string& file);
+    static std::vector<std::string> defaultExtensions = { ".scss", ".sass", ".css" };
```

Comment:
You're right, but isn't it going away by 4 though? Might be another one to conditional out